xfce-extra/xfce4-kbdleds-plugin/xfce4-kbdleds-plugin-0.2.3.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg-utils
+
+DESCRIPTION="A panel plug-in to show state of Caps, Num and Scroll Lock keys"
+HOMEPAGE="https://github.com/oco2000/xfce4-kbdleds-plugin"
```

Comment:
Please also update or add remote-id in metadata.xml.

Reply:
Done.